cockroachcloud/quickstart-trial-cluster.md

```diff
@@ -0,0 +1,183 @@
+4. For **Regions & nodes**, use the default selection of `California (us-west)` region and 3 nodes.
+5. For **Hardware per node**, select `Option 1` (2vCPU, 60 GB disk).
+6. Name the cluster. The cluster name must be 6-20 characters in length, and can include lowercase letters, numbers, and dashes (but no leading or trailing dashes).
+7. Click **Next**.
+8. On the **Summary** page, enter your credit card details.
+
+    {{site.data.alerts.callout_info}}
+    You won't be charged until after your free trial expires in 30 days.
+    {{site.data.alerts.end}}
+
+9. Click **Create cluster**.
+
+Your cluster will be created in approximately 20-30 minutes. Watch this [Getting Started with CockroachCloud](https://youtu.be/3hxSBeE-1tM) video while you wait.
+
+Once your cluster is created, you will be redirected to the **Cluster Overview** page.
+
+## Step 2. Create a SQL user
+
+1. In the left navigation bar, click **SQL Users**.
+2. Click **Add User**. The **Add User** modal displays.
+3. Enter a **Username** and **Password**.
+4. Click **Save**.
+
+## Step 3. Authorize your network
+
+1. In the left navigation bar, click **Networking**.
+2. Click **Add Network**. The **Add Network** modal displays.
+3. From the **Network** dropdown, select **Current Network** to auto-populate your local machine's IP address.
+4. To allow the network to access the cluster's DB Console and to use the CockroachDB client to access the databases, select the **DB Console to monitor the cluster** and **CockroachDB Client to access the databases** checkboxes.
+5. Click **Apply**.
+
+## Step 4. Get the connection string
+
+1. In the top-right corner of the Console, click the **Connect** button. The **Connect** modal displays.
+2. From the **User** dropdown, select the SQL user you created in [Step 2. Create a SQL user](#step-2-create-a-sql-user).
+3. Verify that the `us-west2 GCP` region and `default_db` database are selected.
+4. Click **Continue**. The **Connect** tab is displayed.
+5. Click **Connection string** to get the connection string for your cluster.
+6. Create a `certs` directory on your local workstation.
+7. Click the name of the `ca.crt` file to download the CA certificate to your local machine.
+8. Move the downloaded `ca.crt` file to the `certs` directory.
+
+## Step 5. Run your first query
+
+For this tutorial, we will use the [`movr` workload](../stable/movr.html) to run the first query. On your local machine:
+
+1. [Download the CockroachDB binary](../stable/install-cockroachdb.html):
+
+    <div class="filters clearfix">
+      <button class="filter-button page-level" data-scope="mac">Mac</button>
+      <button class="filter-button page-level" data-scope="linux">Linux</button>
+    </div>
+
+    <section class="filter-content" markdown="1" data-scope="mac">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ curl https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.darwin-10.9-amd64.tgz \
```

Comment:
This is duplicating stuff from the linked install page.  Not end of the world, but it has a cost.  I assume this is intentional.